ios/BetterSocial.xcodeproj/project.pbxproj

```diff
@@ -599,7 +594,7 @@
 				00E356EB1AD99517003FC87E /* Frameworks */,
 				00E356EC1AD99517003FC87E /* Resources */,
 				8FA6F1F4C7A6332D68D1752C /* [CP] Embed Pods Frameworks */,
-				01EE1B5C62B8A36839CBF62D /* [CP] Copy Pods Resources */,
+				8D0A1C5C1044AF97796D96D2 /* [CP] Copy Pods Resources */,
 			);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

This change appears to be a simple replacement of one shell script with another. As long as the new script (`8D0A1C5C1044AF97796D96D2 /* [CP] Copy Pods Resources */`) accomplishes the same tasks as the old one (`01EE1B5C62B8A36839CBF62D /* [CP] Copy Pods Resources */`), this should be fine.



<!-- This is an auto-generated comment by OSS CodeRabbit -->